Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11.6-alpine3.18
+
+ENV PYTHONDONTWRITEBYTECODE=1
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
```

Comment:
Copying the entire context with `COPY . .` can lead to larger Docker images and potential security risks if sensitive files are included. Consider using a more targeted `COPY` command to include only necessary files.